modules/virt-reclaiming-statically-provisioned-persistent-volumes.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reusing-persistent-volumes.adoc
+
+[id="virt-reclaiming-statically-provisioned-persistent-volumes_{context}"]
+= Reclaiming statically provisioned persistent volumes 
+
+Reclaiming a statically provisioned persistent volume (PV) for re-use is dependent on the underlying storage.
+This procedure provides a general approach that might need to be customized depending on your storage.
+
+.Procedure
+
+. Ensure that the reclaim policy of the PV is set to `Retain`:
++
+----
+$ oc get pv <pv_name> -o yaml | grep 'persistentVolumeReclaimPolicy'
+----
+
+.. If the `persistentVolumeReclaimPolicy` is not set to `Retain`, edit the reclaim policy with the following command:
++
+----
+$ oc patch pv <pv_name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+----
+
+. Ensure that no resources are using the persistent volume claim (PVC):
++
+----
+$ oc describe pvc <pvc_name> | grep 'Mounted By:'
+----
++
+Remove any resources that are still using the PVC before continuing.
```

Comment:
yeah, it's not awesome but I have to avoid being prescriptive here because the user might want to delete the object/s, change to different storage, or just drop storage until the new one has been created. 

Sub updated

Reply:
That makes sense! If any of that extra guidance might be helpful to the user instead of just good context for me, consider expanding this statement.